lib/next_rails/bundle_report.rb

```diff
@@ -11,66 +11,9 @@ module BundleReport
     def compatibility(rails_version: nil, ruby_version: nil, include_rails_gems: nil)
       return puts RubyVersionCompatibility.new(options: { ruby_version: ruby_version }).generate if ruby_version
 
-      incompatible_gems = NextRails::GemInfo.all.reject do |gem|
-        gem.compatible_with_rails?(rails_version: rails_version) || (!include_rails_gems && gem.from_rails?)
-      end.sort_by { |gem| gem.name }
+      return unless rails_version
 
-      incompatible_gems.each { |gem| gem.find_latest_compatible(rails_version: rails_version) }
```

Comment:
@arielj could you elavorate a bit on this line, please? why is it needed? how is used?

Reply:
That makes sense, yeah I noticed it was related to finding the latest compatible rails version for the gem, but didn't realize it. thanks! now that I have your attention what do you think about the PR? can you review it please?

Reply:
I updated this one and will continue updating the other good suggestions/fixes that you proposed. Thanks, Ariel!